ops/storage-ddls/cassandra-setup.cql

```diff
@@ -30,30 +29,28 @@ CREATE TABLE sitesettings(
     cogvisionsvctoken text,
     cogtextsvctoken text,
     insertion_time timestamp,
-    PRIMARY KEY (id, sitename)
+    PRIMARY KEY (sitename)
 );
 
-CREATE INDEX ON sitesettings (sitename);
-
 CREATE TABLE streams (
-  pipeline text,
   streamid uuid,
-  connector text,
+  pipelinekey text,
+  pipelinelabel text,
+  pipelineicon text,
+  streamfactory text,
   params frozen<map<text, text>>,
-  PRIMARY KEY ((pipeline), streamid)
+  PRIMARY KEY (pipelinekey, streamid)
 );
 
 CREATE TABLE trustedsources (
-   sourceid text,
+   externalsourceid text,
    sourcetype text,
-   connector text,
+   streamfactory text,
    rank int,
    insertion_time timestamp,
-   PRIMARY KEY ((connector), sourceid, sourcetype)
+   PRIMARY KEY (pipelinekey, externalsourceid, sourcetype, rank)
 );
 
-CREATE INDEX ON trustedsources (rank);
-
 CREATE TYPE computedsentiment (
     pos_avg float,
     neg_avg float
```

Comment:
I wonder if it would be useful to turn the sentiment into a single score in [0(negative)..0.6(neutral)..1(positive)] since that is the sentiment score we get back from the cognitive services API. Also: in the dashboard, we present the sentiment as a single number, so we'd skip having to merge the positive/negative scores into one in the presentation layer.

<img src="https://user-images.githubusercontent.com/1086421/28140306-a9197944-670c-11e7-9785-8f95cafc8140.png" width=200 />